components/landing/TheQiskitCapabilitiesSection/QiskitCapabilityCard.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <article class="qiskit-capability-card">
+    <div class="qiskit-capability-card__container">
+      <div class="qiskit-capability-card__thumbnail">
+        <img class="qiskit-capability-card__thumbnail__media" :src="thumbnailResource">
+      </div>
+      <div class="qiskit-capability-card__copy">
+        <h3 class="qiskit-capability-card__title">
+          {{ title }}
+        </h3>
+        <img class="qiskit-capability-card__thumbnail__mobile" :src="thumbnailResource">
+        <div class="qiskit-capability-card__description">
+          <p v-text="description" />
+          <AppCta
+            v-if="link"
+            v-bind="link"
+            kind="ghost"
+          />
+        </div>
+      </div>
+    </div>
+  </article>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+
+@Component
+export default class QiskitCapabilityCard extends Vue {
+  @Prop({ type: String, required: true }) thumbnailResource!: string
+  @Prop({ type: String, required: true }) title!: string
+  @Prop({ type: String, required: true }) description!: string
+  @Prop({ type: Object, required: false }) link!: any
+}
+</script>
+
+<style lang="scss" scoped>
+.qiskit-capability-card {
+  display: flex;
+
+  @include mq($until: medium) {
+    flex-direction: column;
+  }
+
+  &__container {
+    display: flex;
+
+    @include mq($until: large) {
+      flex-direction: column;
+    }
+  }
+
+  &__copy {
+    padding-right: $spacing-07;
+  }
+
+  &__thumbnail {
+    width: 100%;
+    max-width: 10rem;
+    margin-right: $spacing-07;
+
+    @include mq($until: large) {
+      margin-right: initial;
+      margin-bottom: $spacing-05;
+    }
+
+    @include mq($until: large) {
+      display: none;
+    }
+
+    &__media {
+      display: block;
+      width: 100%;
+      min-width: 10rem;
+
+    }
+
+    &__mobile {
+      display: none;
+      @include mq($until: large) {
+        display: block;
+        width: 100%;
+        max-width: 10rem;
+        margin-bottom: $spacing-05;
+      }
+    }
+  }
+
+  &__title {
+    padding-right: $spacing-07;
+
+    @include mq($until: large) {
+      flex: 0 0 auto;
+      padding-right: 0;
+    }
+
+    @include mq($until: medium) {
+      padding-bottom: 0;
+      margin-bottom: $spacing-05;
+    }
+  }
+
+  &__description {
+    flex: 1;
+
```

Comment:
I believe this has no noticeable effect, but let me know if I'm missing something.

```suggestion
```